ts/features/idpay/initiative/configuration/xstate/services.ts

```diff
@@ -58,29 +62,31 @@ const createServicesImplementation = (
     const data: Promise<IbanListDTO> = pipe(
       response,
       E.fold(
-        _ => Promise.reject(InitiativeFailureType.IBAN_LIST_LOAD_FAILURE),
-        _ => {
-          if (_.status !== 200) {
-            return Promise.reject(InitiativeFailureType.IBAN_LIST_LOAD_FAILURE);
-          }
-
-          // Every time we enroll an iban to an initiative, BE register it as a new iban
-          // so we need to filter the list to avoid duplicates
-          // This workaround will be removed when BE will fix the issue
```

Comment:
why remove this comment?

Reply:
Didn't notice that, maybe a copy-paste fault. Thanks!